replace_template.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# This code will take a template file and change it according to the requirements in the integration-definitions repo
+
+file=$1
+integration=$2
+if grep -q "ParameterGroups" "$file"; then
+    yq eval '.Metadata."AWS::CloudFormation::Interface".ParameterGroups[0].Parameters += "IntegrationId"' -i $file
+fi
+if grep -q "ParameterLabels" "$file"; then
+    yq eval --inplace '.Metadata."AWS::CloudFormation::Interface".ParameterLabels += {"IntegrationId": {"default": "Integration ID"}}' $file
+fi
+
+yq eval --inplace '.Parameters += {"IntegrationId": {"Type": "String",  "Description": "The integration ID to register."}}' $file
+
+echo "  # Used as a bridge because CF doesn't allow for conditional depends on clauses.
+  NonNotifierResourcesAreReady:
```

Comment:
Do we want to support the case in which no resources have a condition? Right now I think we're always creating the  `NonNotifierResourcesAreReady` even though it may be irrelevant in some cases. It's probably fine for now, but let's keep that in mind 👍 .

Reply:
From what i saw every integration have the LambdaFunctionNotificationSubscription resource which has a condition, so i don't think that it will be a problem.